### PR TITLE
Update of HLT menus (91X)

### PR DESCRIPTION
**Pull request by silviodonato**
This PR is an update of the HLT menus.
The GRun menu has been almost completely cleaned up, as asked by the Trigger Coordination.

Other minor changes:
- a fix in HLTrigger/Configuration/python/CustomConfigs.py about L1REPACK
- add a feature to download a specific menu in HLTrigger/Configuration/test/cmsDriver.csh

### CODE REVIEW

**Comment by silviodonato** [comment]
please test

**Comment by silviodonato** [comment]
+1

**Comment by Martin-Grunewald** [comment]
Which IB / release is this PR based on?

**Comment by silviodonato** [comment]
It was based on CMSSW_9_1_0_pre2.

**Comment by Martin-Grunewald** [comment]
hold

**Comment by Martin-Grunewald** [comment]
unhold

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
@silviodonato 
We'd then need an additonal branch taking the role of cms-tsg-storm:for83Xsamples but for 91X (and this PR).

**Comment by silviodonato** [comment]
please test

**Comment by silviodonato** [comment]
I added a fix to hltIntegrationTests running with an external setup (-s option). See https://its.cern.ch/jira/browse/CMSHLT-1228

**Comment by silviodonato** [comment]
+1

File: HLTrigger/HLTfilters/src/HLTSinglet.cc [hunk 1]
```diff
@@ -104,6 +105,7 @@ HLTSinglet<T>::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
   desc.add<double>("MinE",-1.0);
   desc.add<double>("MinPt",-1.0);
   desc.add<double>("MinMass",-1.0);
+  desc.add<double>("MinEta",-1.0);
```
Comment (davidlange6) [inline HLTrigger/HLTfilters/src/HLTSinglet.cc]: Hi @silviodonato - MinAbsEta is likely a less confusing name for this parameter in the long term.

**Comment by davidlange6** [comment]
+1

**Pull request merged**